tests/test_config/test_correct_year.py

```diff
@@ -39,7 +40,7 @@ def test_correct_year(expect_date):
     cfg = Config({'copyright': copyright_date}, {})
     assert cfg.copyright == copyright_date
     correct_copyright_year(None, cfg)  # type: ignore[arg-type]
-    if expect_date:
+    if expect_date and expect_date <= '2009':
```

Comment:
```suggestion
    if expect_date and expect_date <= LT.tm_year:
```

...could express the intent more clearly?

Reply:
I've pushed a change to do that; I'm not certain it's an improvement, so feel free to revert.